agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/metrics/metrics.go

```diff
@@ -42,8 +42,8 @@ type Entry interface {
 	// for reporting numerical values related to any operation, for instance the data transfer
 	// rate for an image pull operation.
 	WithGauge(value interface{}) Entry
-	// Done makes a metric operation as complete. It records the end time of the operation
-	// flushes the metrics to a persistent store.
+	// Done makes a metric operation as complete. It records the end time of the operation and
```

Comment:
I'm assuming changes here and on L46 are the result of vendor file not being updated in a previous PR that changes the `ecs-agent/metrics/metrics.go` file in ecs-agent module and then running `go mod tidy` and `go mod vendor` here. If that's the case, can we mention this in the PR description?

Reply:
Hmm, yeah looks like it. Updated the PR description